blazar.yaml

```diff
@@ -0,0 +1,26 @@
+buildpack:
+  host: git.hubteam.com
+  organization: HubSpotProtected
+  repository: Blazar-Buildpack-Java
+  branch: v2-prebuilt
+
+stepActivation:
+  uploadAndNotify:
+    branches: ['master']
+
+steps:
+  - description: "Test across all supported Scala Versions"
+    commands:
+      - echo "SBT_OPTS='-Xms512M -Xmx1024M -Xss2M -XX:MaxMetaspaceSize=1024M' +test"
+      - SBT_OPTS="-Xms512M -Xmx1024M -Xss2M -XX:MaxMetaspaceSize=1024M" +test
```

Comment:
This command tests across all versions of Scala in `crossScalaVersions` in `build.sbt`. Tbh, I've never used anything besides the 2.12 version that is the default, is there really any benefit in supporting multiple versions? 

The extra memory is needed for the cross-version testing or else it OOMs. If we decide not to do cross version testing, then we can omit those args